content/en/docs/topics/kubernetes_apis.md

```diff
@@ -123,15 +123,20 @@ Alternatively, you can follow these manual steps to perform an update of the API
 versions of a release manifest:
 
 - Get the name of the latest deployed release:
-`kubectl get secret -l owner=helm,status=deployed,name=<release_name> --namespace <release_namespace> | awk '{print $1}' | grep -v NAME`
+  - Secrets backend: `kubectl get secret -l owner=helm,status=deployed,name=<release_name> --namespace <release_namespace> | awk '{print $1}' | grep -v NAME`
+  - ConfigMap backend: `kubectl get configmap -l owner=helm,status=deployed,name=<release_name> --namespace <release_namespace> | awk '{print $1}' | grep -v NAME`
 - Get latest deployed release details:
-`kubectl get secret <release_version_secret_name> -n <release_version_namespace> -o yaml > release.bak`
+  - Secrets backend: `kubectl get secret <release_version_secret_name> -n <release_version_namespace> -o yaml > release.bak`
+  - ConfigMap backend: `kubectl get configmap <release_version_secret_name> -n <release_version_namespace> -o yaml > release.bak`
```

Comment:
`<release_version_configmap_name>` instead of `<release_version_secret_name>`?
So actually `<release_configmap_name>`(without the word "version")

Reply:
Agree, it is convoluted. Changed.